organizing_hub/templates/calls/callcampaign_detail.html

```diff
@@ -124,18 +127,29 @@ <h3>Callers</h3>
       <p>
         <strong>{{ campaign.callers.all|length }}</strong>
         callers on this campaign.
-        <a href="todo">
-          View
+        <a href="{% url 'organizing-hub-call-campaign-update' campaign.uuid %}">
+          Edit
         </a>
       </p>
 
+      <p>
+        Callers will be able to login and make calls using the email address
+        listed after creating an account. You may add or remove callers at any
```

Comment:
I was a bit confused by language `Callers will be able to login and make calls using the email address listed after creating an account.` Suggest something more like `Callers will be able to login and make calls after creating an account using the email address specified here.`